setup.py

```diff
@@ -58,7 +58,7 @@ def version():
     author_email="fiaas@googlegroups.com",
     version=version(),
     packages=find_packages(exclude=("tests",)),
-    zip_safe=False,
+    zip_safe=True,
```

Comment:
This breaks the frontend application as the templates cannot be loaded.

Reply:
That's weird. It shouldn't... :thinking: 
:man_shrugging: Easy enough to flip the flag back though, I'll fix.

Reply:
#5